modules/districtmDMXBidAdapter.js

```diff
@@ -41,7 +43,7 @@ export const spec = {
               nBid.height = nBid.h || height;
               nBid.ttl = 300;
               nBid.mediaType = bid.mediaTypes && bid.mediaTypes.video ? 'video' : 'banner';
-              if (nBid.mediaType) {
+              if (nBid.mediaType && nBid.mediaType === 'video') {
```

Comment:
not a big deal, but just noticed that this could just be
```if (nBid.mediaType === 'video') {```
don't need the `nBid.mediaType` part first.